content/installation/net/configuration/YAML-properties-net.md

```diff
@@ -26,6 +27,28 @@ The *contrast_security.yaml* file should be placed on the file system using one
 * Specify the path to the YAML file with the environment variable `CONTRAST_CONFIG_PATH`.
 * Place the *contrast_security.yaml* file in the data directory specified during agent install. (The default location is * %ProgramData%\Contrast\dotnet\*. As a result, the default file path would be *%ProgramData%\Contrast\dotnet\contrast_security.yaml*.)
 
+## Using Environment Variables
+Every configuration option supported by the *contrast_security.yaml* file can also be specified using environment variables. 
+
+Environment variable names are derived from the yaml path by replacing path segment delimiters (`.`) with double underscores (`__`) and prefixing the result with `CONTRAST__` e.g. `server.name` becomes `CONTRAST__SERVER__NAME` while `api.api_key` becomes `CONTRAST__API__API_KEY`.
```

Comment:
We need to make sure that these underscores are rendered correctly. Github markdown renders correctly, but the markdown engine we are using for docs doesn't handle them well.

I think this is fine as we are using the ticks.